hr_timesheet_sheet_worked_time/models/hr_timesheet_sheet_sheet.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+from odoo import models, fields, api
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class HrTimesheetSheet(models.Model):
+    _inherit = 'hr_timesheet_sheet.sheet'
+
+    standard_work_hours = fields.Float(
+        compute='_compute_standard_work_hours',
+        string="Standard Work Hours",
+    )
+    expected_work_hours = fields.Float(
+        compute='_compute_expected_work_hours',
+        string="Expected Work Hours",
+    )
+    overtime_hours = fields.Float(
+        compute='_compute_overtime_hours',
+        string="Overtime Hours",
+    )
+    holiday_hours = fields.Float(
+        compute='_compute_holiday_hours',
+        string="Holiday Hours",
+    )
+
+    @api.depends(
```

Comment:
Do we need `depends` since `holiday_hours` is not a stored field? May I know your intention?
Please add `@api.multi`

Reply:
the main reason is to use depends: the user must know if any changes on the attendance of calendar id and day week then Holidays hours must also be updated.
and the for @api.multi will load the value when we open the form view of the timesheet.

So it is Let me add api.multi

Reply:
If those fields are non-stored, then we do not need `api.depends`